effectiveOfficeBackend/src/main/kotlin/office/effective/features/user/repository/UserRepository.kt

```diff
@@ -87,5 +87,32 @@ class UserRepository {
         return UserTagModel(tag.id, tag.name)
     }
 
+    fun updateUser(model: UserModel): UserModel {
+        model.id ?: throw UserNotFoundException("No id in the model")
+        if (!existsById(model.id!!)) {
+            throw UserNotFoundException("User ${model.fullName} with id:${model.id} does not exists")
+        }
+        var ent = db.users.find { it.id eq model.id!! }
```

Comment:
Why are you treating this variable as nullable if you did a check on the first line?

Reply:
Solved